build/Dockerfile.client

```diff
@@ -1,4 +1,4 @@
-ARG GOLANG_IMAGE_VERSION=golang:1.18.3-alpine3.16
+ARG GOLANG_IMAGE_VERSION=golang:1.19-alpine3.16
```

Comment:
nit: why not latest of [1.20.3](https://hub.docker.com/layers/library/golang/1.20.3/images/sha256-9413aaefd182f9499f7e17b7400709c34eb1edac9ca71c37cdc73a63691c5f95?context=explore)?

Reply:
My intention was only to synchronize the current version of go that we're using, which is 1.19.x. The reason was an error produced by go-libp2p-kad-dht or go-libp2p-pubsub IIRC, complaining about needing go 1.19 or later.

I think we anticipate that the 1.20 migration may require some effort, see #588.

Reply:
I'm usually weary of upgrading language versions & business logic in the same PR (harder to revert, things can break, etc...) but assuming you're confident its safe in this scenario?

Reply:
`Dockerfile.client` (this file) is the image that provides the debug CLI in docker compose localnet. The other dockerfile that I modified ([`Dockerfile.localdev`](https://github.com/pokt-network/pocket/blob/main/build/Dockerfile.localdev)) is also a compose localnet only image (which includes `dlv` debugger :call_me_hand:). See the [docker-compose.yaml](https://github.com/pokt-network/pocket/blob/main/build/deployments/docker-compose.yaml). I compared these to what we're using in production ([`Dockerfile.debian.prod`](https://github.com/pokt-network/pocket/blob/main/build/Dockerfile.debian.prod)) which is already using go 1.19. See [`build-images` CI job](https://github.com/pokt-network/pocket/blob/main/.github/workflows/main.yml#L149) for how `Dockerfile.debian.prod` is used by CI to build production the image.